src/components/_checkbox.scss

```
@@ -6,144 +6,140 @@ category: Components
 ---
 
 オプションのオン/オフ切り替えなどをする際に使用します。
-オプションから選択できる項目が２つ以上の場合には(ただし任意選択を除く)、Checkboxの代わりにRadioを使用しましょう。
-
-```example
-<label class="ncgr-checkbox">
-  <input type="checkbox" name="hoge">
-  <span>
-    <i class="fas fa-check"></i>
-    甲賀流
-  </span>
-</label>
-
-<label class="ncgr-checkbox">
-  <input type="checkbox" name="hoge">
-  <span>
-    <i class="fas fa-check"></i>
-    伊賀流
-  </span>
-</label>
+List Viewのleadingやtrailingなどの中に入れて使用することもできます。
```

いまいまその使い方しかないので決め打ちでよさそう。

```suggestion
List View の Leading や Trailing の中に入れて使用します。
```